scripts/autotracking.lua

```diff
@@ -403,30 +495,59 @@ function updateItemsFromInventory(segment)
   end
 
   -- Loop through the inventory, determine which key items the player has found
+  local objectives = 0
   for i=0,0xF1 do
     local item = segment:ReadUInt8(0x7E2400 + i)
     -- Loop through the table of key items and see if the current
     -- inventory slot maches any of them
-    for _,v in pairs(KEY_ITEMS) do
-      if type(v.value) == "number" then
-        if item == v.value then
-          v.found = true
-        end
-      elseif type(v.value) == "table" then
-        -- Loop through possible IDs for items with more than one
-        -- Not used since the Masamume/Grand Leon change, but leaving this in
-        -- in case it's needed in the future.
-        for _, v2 in pairs(v.value) do
-          if item == v2 then
+    if item == 0x1B then
```

Comment:
This 0x1B appears to be the single "objectives met" item, and is not tracking the objectives themselves.